### PR TITLE
45 round controllers

### PR DESCRIPTION
**Pull request by polarbear23**

### CODE REVIEW

File: src/server/routes/competition.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 const express = require('express');
-const { createCompetition, getCompetitionById, } = require('../controllers/competition');
 const passport = require('passport');
 
+const { createCompetition, getCompetitionById, } = require('../controllers/competition');
+
+
```
Comment (bravint) [inline src/server/routes/competition.js:6]: please remove whitespace

File: src/server/routes/round.js [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
-} = require('../controllers/round');
+const passport = require('passport');
+
+const { createRound, getAllRoundsBySeasonId } = require('../controllers/round');
 
 const router = express.Router();
 
-router.post('/round', round);
+router.post('/', passport.authenticate('jwt', { session: false }), createRound);
+router.get('/', passport.authenticate('jwt', { session: false }), getAllRoundsBySeasonId);
```
Comment (bravint) [inline src/server/routes/round.js:9]: not sure get requests need authorization

**Review by nathanSamosa — APPROVED** [review]

**Review by bravint — APPROVED** [review]
looks good

File: src/server/routes/round.js (hunk 2 above)
Comment (polarbear23) [reply src/server/routes/round.js:9]: ok will remove it for now

File: src/server/routes/competition.js (hunk 1 above)
Comment (polarbear23) [reply src/server/routes/competition.js:6]: ok removed

**Review by molloyconor511 — APPROVED** [review]

**Pull request merged**